Tiltfile

```diff
@@ -0,0 +1,21 @@
+# Version
+version_settings(constraint='>=0.22.2')
+
+load('ext://dotenv', 'dotenv')
+dotenv('tilt.env')
```

Comment:
do we have a default tilt.env for development?

Reply:
`tilt.env` is not required for `tilt`.  

When we use `kind` with `tilt`, the tilt expects the cluster to be ready. So  I added `make tilt-setup i=ipv4 b=nft` for creating a cluster. When we create a kind cluster, We need to know  the `ip family` to decide `podSubnet` and  `serviceSubnet`

When we run `tilt up` or `make tilt-up`, We need to generate kpng DaemonSet `yaml` and build image. then deploy into the kind cluster.  To generate daemonset yaml, We need to know what the ip family and backend we provided when setup the cluster. So I use `tilt.env` file to store the environment variables including backend, and ip-family(`tilt.env` will be created when we run `make tilt-setup`).  Then we can read it inside tilt session. 

If we need to change the backend, We can change it in `tilt.env` file. It'll trigger an update to `tilt`.

There may be a better approach than this 🙂

Reply:
what about a tilt.dev.example and say in the readme you can copy it over. so the user has a template (unless make tilt-up) is generating this automatically

Reply:
+1

Reply:
@knabben @ksankeerth just wondering if the Tiltfile should be inside the tilt dir?

Reply:
@dougsland I think it's a good suggestion. Mostly, the users don't need to change Tiltfile. I'll move to tilt dir.